app/script/view_model/bindings/CommonBindings.js

```diff
@@ -191,8 +188,9 @@ ko.bindingHandlers.resize = (function() {
       const params = ko.unwrap(valueAccessor()) || {};
       triggerValue = params.triggerValue;
       syncElement = document.querySelector(params.syncElement);
+      throttledResizeTextarea = _.throttle(resizeTextarea, 100, {leading: !params.delayedResize});
```

Comment:
There might a problem here, I should create the throttled version in the `init` function instead

Reply:
Yeah, that actually makes a lot of sense.